scripts/py_matter_yamltests/matter_yamltests/parser.py

```diff
@@ -1223,7 +1223,8 @@ def _config_variable_substitution(self, value):
             # But some other tests were relying on the fact that the expression was put 'as if' in
             # the generated code and was resolved before being sent over the wire. For such
             # expressions (e.g 'myVar + 1') we need to compute it before sending it over the wire.
-            tokens = value.split()
+            delimiter_regex = "(\ |\+|\-|\*|\/|\%)"
```

Comment:
is there any way for us to unit test some bits of these? It seems useful if we could have unit tests on yaml test parsing and interpreting.

Reply:
We have some - maybe in test_yaml_parser.py?

Reply:
Added